Project-Kampong/kampong-frontend
# Netlify integration

# Types of changes in this PR

- [ ] Breaking change of an existing feature
- [ ] Refactor/Performance enhancements
- [ ] Infra/Config changes

# Overview (attach screenshots if necessary)

- Add netlify badge to README
- Also, to test netlify CI with PRs
- Add client-side routing to work on netlify (see https://scotch.io/tutorials/deploying-an-angular-app-to-netlify)
- See netlify build live on 
1. https://kampong.netlify.app (current `master` branch)  
2. https://5f8c7d33b869ee0008aa175c--kampong.netlify.app/home (current PR)
